src/zif_abapgit_definitions.intf.abap

```diff
@@ -8,6 +8,12 @@ INTERFACE zif_abapgit_definitions
     ty_bitbyte TYPE c LENGTH 8 .
   TYPES:
     ty_sha1    TYPE c LENGTH 40 .
+  TYPES: BEGIN OF ty_commit_value_tab,
```

Comment:
suggest moving this to the method or class instead, it is unlikely to be reused

Reply:
both TYPES

Reply:
ty_sha1 is being used several times in different classes later so we should let it stay in definitions. Even if it's unlikely to be reused isn't it a better approach to use only one place (in this project interface definitions) to define all types ever supposed to be used to avoid redundancy and push transparency about available types?

Reply:
zif_abapgit_definitions is already too large and must be structured and cleaned up in the future, 

I don't want to add types that are not needed as global, its always possible to move it to global types if needed.